### ISSUE TITLE
Inline outer function

### ISSUE THREAD

**Issue by bestouff** [llogiq/momo#3]
The outer function should probably be inlined.

**Comment by llogiq** [comment]
The current code should do this. However, if the outer function already has an inline annotation, it will get two of them. I should add a check, which is why I'm leaving this open.

**Comment by bestouff** [comment]
Good !  
Isn't your problem just a case of "if it hurts, don't do it" ?

**Comment by llogiq** [comment]
No, because we clone the original function definition's attributes to not interfere with e.g. docs. But if the original function was already inlined, we need to inline the inner function instead.
